debridge-finance/debridge-solana-sdk
# chore(check_claiming): Added check claiming function

## What?
Created debirdge-sdk library that provide functionality for integration with debridge infrastructure.

## Why?
It needs to integrators to simply connect their protocol to debridge infrastructure.

## How?
- Added ```send_via_debridge``` function that allow send some token or message from debridge infrastructure. 
- Added ```check_claiming``` function that allow check debridge claim call by submission account checking. This function checks that it is executed by debridge protocol ```execute_external_call``` function. Also checks submissions params and native sender

## Testing?
- Created example program for testing
- Send through debdrige with example program
- Check that example program function executed by ```execute_external_call``` function and checks submissions params and native sender correctness 

## Comments
